fedmsg_meta_fedora_infrastructure/tests/releng.py

```diff
@@ -108,7 +376,6 @@ class TestTwoWeekAtomicComplete(Base):
                 "name": "Fedora-Cloud-Atomic-Raw"
             }
         }
-    }
-
+}
 
 add_doc(locals())
```

Comment:
The 2 empty lines at the end may be a pep8/flake8 requirement :)

Reply:
Ah yes, fixed it :)